tests/jlm/llvm/frontend/llvm/LoadTests.cpp

```diff
@@ -25,7 +25,8 @@ LoadConversion()
   std::unique_ptr<Module> llvmModule(new Module("module", context));
 
   auto int64Type = Type::getInt64Ty(context);
-  auto pointerType = Type::getInt64PtrTy(context);
+  // ADDRESS_SPACE_GENERIC == 0
```

Comment:
@sjalander Where does this comment come from all in a sudden?

Reply:
It's outdated